template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -6,8 +6,9 @@
       "longName": "target-framework-override",
       "shortName": ""
     },
-    "UseLocalDB": {
-      "longName": "use-local-db"
+    "EFProvider": {
+      "longName": "ef-provider",
+      "shortName": "ef"
```

Comment:
Feels like these should be something like `db-provider` and `db` so they can be reused across templates regardless of data access technology.